src/js/components/StarRating/StarRating.js

```diff
@@ -4,10 +4,9 @@ import { Star } from 'grommet-icons/icons/Star';
 import { StarOutline } from 'grommet-icons/icons/StarOutline';
 import { RadioButtonGroup } from '../RadioButtonGroup';
 
-const StarRating = ({ ...rest }) => {
+const StarRating = ({ value: defaultValue, ...rest }) => {
```

Comment:
Hello,

I tested it, and it works fine for a default value, but I think we need to be able to synch the value with the radioButtonGroup value. 

For example, currently, if I try to use StarRating into a form, it is not working. 

This solution will allow me to put it manually, but this doesn't seem like the dream solution.


What about leaving the `RadioButtonGroup` to deal with the value and just fetching the value from the context, to deal with displaying?

```
const [value, _] = formContext.useFormInput({
      name,
    });
```

Like this we don't deal with two states that are actually the same value.

I might be missing a lot of aspect, so please don't avoid telling me where I am mistaking if this is the case.